packages/@expo/metro-config/src/transform-plugins/__tests__/import-export-plugin.test.ts

```diff
@@ -198,24 +196,21 @@ it('imports multiple members from another module and export them in separate sta
 
   const expected = `
     Object.defineProperty(exports, '__esModule', {value: true});
-
-    var _bar = require('bar');
-    var bar = _$$_IMPORT_DEFAULT('bar');
-
-    exports.bar = bar;
     Object.defineProperty(exports, "foo", {
       enumerable: true,
       get: function () {
         return _bar.foo;
       }
     });
-
     Object.defineProperty(exports, "baz", {
       enumerable: true,
       get: function () {
         return _bar.baz;
       }
     });
+    var _bar = require('bar');
+    var bar = _$$_IMPORT_DEFAULT('bar');
+    exports.bar = bar;
```

Comment:
unsure about this one. If this isn't hoisted, and we enter a cycle, this assignment may not be evaluated in time, so will likely also need to be a live binding, i.e. if we have live bindings enabled, I believe, every `exports.` will need to be a live binding? Obviously impossible for `export *` cases, so unsure what's done in that case, but this feels like there's a failure case here if it's accessed in the top-level in a cycle somewhere